/rexrov_formation_lifecycle_node:
  ros__parameters:
    Lambda: 0.7
    Gamma: 0.01
    C1: 0.4
    C2: 1.0
    rot_p: 5.0